src/safe_balances.test.js

```diff
@@ -0,0 +1,76 @@
+import { assertTextPresent, clickElement, clickSomething, getInnerText } from '../utils/selectorsHelpers'
+import { getEnvUrl, initWithWalletConnected } from '../utils/testSetup'
+import config from '../utils/config'
+import { safeBalancesPage } from '../utils/selectors/safeBalancesPage'
+
+/*
+Safe Balances
+-- Enters into the Safe Balances component
+-- check the currency dropdown
+*/
+
+let browser
+let gnosisPage
+
+const { TESTING_SAFE_ADDRESS } = config
+
+beforeAll(async () => {
+  const context = await initWithWalletConnected(true)
+  browser = context[0]
+  gnosisPage = context[2]
+}, 60000)
+
+afterAll(async () => {
+  await gnosisPage.waitForTimeout(2000)
+  const pages = await browser.pages()
+  await Promise.all(pages.map((page) => page.close()))
+  await browser.close()
+})
+
+describe('Safe Balances', () => {
+  test('Safe Balances', async () => {
+    console.log('Safe Balances')
+
+    // goes to the Safe Balance page
+    console.log('Enters into the Safe Balances component')
+    const safeBalancesUrl = `${getEnvUrl()}app/rinkeby/safes/${TESTING_SAFE_ADDRESS}/balances`
+    await gnosisPage.goto(safeBalancesUrl)
+
+    // USD currency by default
+    console.log('USD currency by default')
```

Comment:
Does it make sense to duplicate the console.logs in comments?
Comments look a bit nicer in the editor, I guess? One of the suggestions for the coverage script was to use comments instead of logs.

Reply:
yes, I added them because they look a bit nicer in the editor. I will remove the duplicated ones 👍